lib/codeowners/config.rb

```diff
@@ -40,10 +40,34 @@ def default_owner=(name)
       @git.config('user.owner', name)
     end
 
+    def default_organization
+      config_org = @git.config('user.organization')
+      return config_org.strip unless config_org.nil? || config_org.strip.empty?
+
+      parse_organization_from_origin
+    end
+
+    def default_organization=(name)
+      @git.config('user.organization', name)
+    end
+
     def to_h
       {
-        default_owner: default_owner
+        default_owner: default_owner,
+        default_organization: default_organization
       }
     end
+
+    protected
+
+    def parse_organization_from_origin
+      origin_url = @git.config('remote.origin.url')
+      return '' if origin_url.nil? || origin_url.strip.empty?
+
+      org_regexp = origin_url.match(%r{^https?://.+?/(?<org>.+?)/}) || origin_url.match(%r{:(?<org>.+?)/})
```

Comment:
I haven't tested manually but I guess we can use `|` from regex and combine both:

```suggestion
      org_regexp = origin_url.match(%r{^https?://.+?/(?<org>.+?)/|:(?<org>.+?)/})
```

Reply:
works like a charm, nice suggestion!